validmind/tests/model_validation/sklearn/RegressionModelsPerformanceComparison.py

```diff
@@ -57,7 +57,7 @@ class RegressionModelsPerformanceComparison(Metric):
 
     name = "models_performance_comparison"
 
-    name = "regression_errors"
+    name = "regression_errors_comparison"
```

Comment:
There are two name variables define.  We need to remove one of `name`  variable.

Reply:
Oops! Fixed.